.github/workflows/binary-builds.yml

```diff
@@ -164,63 +168,40 @@ jobs:
         uses: actions/setup-node@49933ea5288caeca8642d1e84afbd3f7d6820020 # v4.4.0
         with:
           node-version: ${{ matrix.node }}
-      - name: Install pnpm
-        if: ${{ matrix.os != 'linux' }}
-        run: |
-          npm install --global pnpm
       - name: Get user info
         id: user_info
         if: ${{ matrix.os == 'linux' }}
         run: |
           echo -n "user=$(id -u):$(id -g)" > $GITHUB_OUTPUT
-      - name: Build gnu libc for linux
+      - name: Build for linux
         uses: addnab/docker-run-action@4f65fabd2431ebc8d299f8e5a018d79a769ae185 # v3
-        if: ${{ matrix.os == 'linux' && matrix.libc == 'gnu' }}
+        if: ${{ matrix.os == 'linux' }}
         with:
           image: ${{ matrix.image }}
-          options: -v ${{ github.workspace }}:${{ github.workspace }}
+          options: |
+            -e NVM_NODEJS_ORG_MIRROR=${{ matrix.node-download-url }}
+            --network host
+            -v ${{ github.workspace }}:${{ github.workspace }}
+            -w ${{ github.workspace }}
           run: |
-            # Setup debian builder
-            apt update
-            apt install -y curl
+            # Setup builder
+            ${{ matrix.prepare }}
 
             # Install nvm
-            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.40.3/install.sh | bash
+            curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/${{ env.NVM_VERSION }}/install.sh | bash
```

Comment:
@setchy Is there a way we can have renovate to check for updates of nvm? If necessary, the EnvVar can be put back in this line, I didn't notice we're only using it once after this PR.